src/ManifoldMuseum.jl

```diff
@@ -3,24 +3,91 @@ module ManifoldMuseum
 import Base: isapprox, exp, log
 import LinearAlgebra: dot, norm
 
+export Manifold
+export dimension,
+    distance,
+    dot,
+    exp!,
+    geodesic,
+    log!,
+    norm,
+    retract,
+    retract!,
+    typical_distance,
```

Comment:
I know it was already defined, but what is the meaning and use case of `typical_distance`?

Reply:
In Manopt (the Matlab one) it's used as an initial guess for trust region based methods and we could also rename it to refer to injectivity radius or something. I am using that number as a distance we definetly log is still defined.

Reply:
This, I think, is from Manopt/Manopt.jl: https://www.manopt.org/tutorial.html#manifoldslibrary .

Reply:
I see. Yeah, I think `injectivity_radius(M::Manifold, x)` makes more sense, since it's point-dependent (e.g. on the upper hemisphere, points near the boundary have a lower injectivity radius than points far from the boundary).